config/gremlin-server-ci.yaml

```diff
@@ -15,4 +15,5 @@ scriptEngines: {
 
 serializers:
   - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV1d0, config: { serializeResultToString: true }}
-ssl: { enabled: true}
+ssl: { enabled: false}
```

Comment:
nit: missing space at the closing bracket?

Reply:
Yes, will fix it.